msteams-platform/sbs-gs-workflow-bot.yml

```diff
@@ -110,53 +111,54 @@ items:
+        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-workspace-folder.png" alt-text=" Sereenshots shows the selection to add workspace folder.":::
        
     1. Enter a suitable name for your app.
               
-        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-app-name.png" alt-text="enter application name":::
+        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-app-name.png" alt-text="Screenshots shows  to enter application name.":::
     
     1. Select **Enter**.
         
     1. The workflow bot is created in a few seconds, and you can see the scaffolding.
         
-        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding1.png" alt-text="your app is created with the scaffolding visible in the side panel":::
+        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding1.png" alt-text="Screenshot shows your app is created with the scaffolding visible in the side panel.":::
     
     1. Select **Run and Debug** :::image type="icon" source="../msteams-platform/assets/images/teams-toolkit-v2/run-debug-icon.png"::: icon from the Visual Studio Code **Activity Bar**.
     
     1. Select **Debug (Edge)** or **Debug (Chrome)** from the dropdown list.
     
-       :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-local-debug1.png" alt-text="your app is created with the scaffolding visible in the side panel":::    
+       :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-local-debug1.png" alt-text="Screenshot shows your app is created with the scaffolding visible in the side panel.":::    
     
     1. Select **Start Debugging**.
     
     1. Select **Add**.
-        :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-add-app.png" alt-text="select add to add your workflow bot app":::
+
+     :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-add-app.png" alt-text="Screenshot shows the selction to add your workflow bot app.":::
     ### Take tour of source code
     Teams Toolkit provides components for building an app. After creating the project, you can view the project folders and files in the Explorer area of Visual Studio Code for JavaScript/TypeScript. 
     
-    :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding2.png" alt-text="select add to add your workflow bot app":::
+    :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding2.png" alt-text="Screenshot shows the selction to add your workflow bot app.":::
     
     
     The created bot is a normal TeamsFx project that contains following folders:
```

Comment:
The folder structure is out-of-date in the latest version workflow app.
Please refer to the latest [README.md]( https://github.com/OfficeDev/TeamsFx/tree/main/templates/scenarios/js/workflow#whats-included-in-the-template) to update it.